spacy/lang/grc/punctuation.py

```diff
@@ -0,0 +1,52 @@
+from ..char_classes import LIST_PUNCT, LIST_ELLIPSES, LIST_QUOTES, LIST_CURRENCY
+from ..char_classes import LIST_ICONS, ALPHA_LOWER, ALPHA_UPPER, ALPHA, HYPHENS
+from ..char_classes import CONCAT_QUOTES
+
+_prefixes = (
+    [
+        "〈",
+        "†",
+        "⟦",
+        "⸏",
```

Comment:
Yes, it could be useful to add them to the char_classes and this would render the punctuation file unnecessary. These characters are used in many historical languages to indicate the type of reconstruction. So they will work also for old documents in many European languages, Latin, Akkadian, Sumerian, and other ancient languages.